.github/workflows/linuxPullRequest.yml

```diff
@@ -2,6 +2,10 @@ name: CI Linux PR
 on:
   pull_request:
     branches: [ main ]
+  workflow_dispatch:
+    inputs:
+      tags:
+        description: 'Test scenario tags'
```

Comment:
I would find it better to name it 'Log level' and maybe you can provide 'debug' as the default value